FWCore/Framework/interface/EventSetupRecord.h

```diff
@@ -66,166 +67,160 @@ using the 'setEventSetup' and 'clearEventSetup' functions.
 
 // forward declarations
 namespace cms {
-   class Exception;
+  class Exception;
 }
 
 class testEventsetup;
 class testEventsetupRecord;
 
 namespace edm {
-   class ESHandleExceptionFactory;
```

Comment:
please return this file back to its original spaces to make review easier.

Reply:
Will do.